routes/login.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.post('/', (req, res) => {
+  db.getUser(req.body).then(data => res.send(data))
```

Comment:
It should be GET request.

Read:
- http requests
- http request headers
- https status

Reply:
Express error handling
https://expressjs.com/en/guide/error-handling.html

Reply:
fixed, but I do not think this option is correct

Reply:
done through headers